analyses/fusion_filtering/02-fusion-filtering.R

```diff
@@ -179,8 +175,16 @@ saveRDS(QCFiltered,paste0(opt$outputfile,"_QC_filtered.RDS"))
 
 # load expressionMatrix RDS for expression based filtering for less than given threshold
 expressionMatrix<-readRDS(expressionMatrix)
-expressionMatrix <- cbind(expressionMatrix, colsplit(expressionMatrix$gene_id, pattern = '_', names = c("EnsembleID","GeneSymbol")))
-
+# find the list of cohorts and sample type of interest
+matched_samples <- read.delim(clinicalFile, header = TRUE, sep = "\t", stringsAsFactors = FALSE) %>%
```

Comment:
I get an error that `object 'clinicalFile' not found`, did you want to read the arg as `clinicalFile<-opt$clinicalFile` before this line?

Reply:
Yes! Fixed that!